Automattic/pocket-casts-android
# [Project] Up Next Shuffle - Update Shuffle Button for Free users

## Description
- This updates the shuffle button for free users
- Context: p1730077568183099-slack-C07T08CTND9
- I will display the paywall when tapping on shuffle button for free users in another PR
- I asked here if we are going to use the same gold color for all themes: p1730220951890729/1730077568.183099-slack-C07T08CTND9

Fixes #3124

## Testing Instructions
1. Run the App with free user
2. Play an episode
3. Open Up next
4. ✅  Ensure you see up next shuffle painted as gold color
5. Log in with a plus user account
6. ✅  Ensure you see up next shuffle is not painted with gold color

## Screenshots or Screencast 

| dark-contrast       | dark                  | extra-dark           |
|---------------------|-----------------------|-----------------------|
| ![dark-contrast](https://github.com/user-attachments/assets/68bf4392-013d-4c9a-a311-b40a0a81152f#width=400) | ![dark](https://github.com/user-attachments/assets/b8b1dc23-3c97-4804-9e5e-c4e9449f9ea0#width=400) | ![extra-dark](https://github.com/user-attachments/assets/e7bd997b-edc2-4be5-a659-692b93e84fb2#width=400) |

| indigo              | light-contrast        | light                |
|---------------------|-----------------------|-----------------------|
| ![indigo](https://github.com/user-attachments/assets/4dca282c-436f-4dff-b7cb-61bd31f6918c#width=400) | ![light-contrast](https://github.com/user-attachments/assets/5f83bb39-2a4c-48bc-adb3-406dc2606c9e#width=400) | ![light](https://github.com/user-attachments/assets/80306597-7cfe-4054-9781-ed599eeebf83#width=400) |

| Rose |
|--------|
| <img src="https://github.com/user-attachments/assets/35d65a7d-88fe-4d06-bbb3-be1a88bec834" height="500">
 | 



## Checklist
 
#### I have tested any UI changes...
